cmd/daprd/components/binding_azure_openai.go

```diff
@@ -0,0 +1,25 @@
+//go:build allcomponents || stablecomponents
```

Comment:
```suggestion
//go:build allcomponents
```

This component is not stable :)